chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetProps.ts

```diff
@@ -51,6 +52,7 @@ export interface ILiveChatWidgetProps {
     preChatSurveyPaneProps?: IPreChatSurveyPaneProps;
     proactiveChatPaneProps?: IProactiveChatPaneStatefulProps;
     reconnectChatPaneProps?: IReconnectChatPaneStatefulProps;
+    startChatErrorPaneProps?: IStartChatErrorPaneProps;
```

Comment:
We might need to add this in dummyDefaultProps
https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts

Reply:
+1

Reply:
Added